roles/irods_icat/tasks/setup_pam.yml

```diff
@@ -48,3 +48,14 @@
     group: root
     mode: 04755
   when: not ansible_check_mode
+
+
+- name: Ensure OIDC authentication is present
+  become_user: '{{ irods_service_account }}'
+  become: yes
+  template:
+    src: oidc.py.j2
+    dest: /var/lib/irods/msiExecCmd_bin/oidc.py
+    owner: irods
+    group: irods
+    mode: 0644
```

Comment:
This task could be conditional? E.g. `when: oidc_active is defined and oidc_active is sameas true`